docs/rules/expiring-todo-comments.md

```diff
@@ -0,0 +1,56 @@
+# Add conditions to TODO comments to make them error
+
+Makes possible to pass arguments to TODO and FIXME comments to trigger errors.
+
+Currently supporting:
+
+* `[YYYY-MM-DD]` to define a due date.
+* `[> 1]` or `[>= 2]` to expire at some version (from package.json).
```

Comment:
I think we should require there to be no whitespace here to reduce ambiguity. Making it stricter also leaves the door open for other conditions in the future.
```suggestion
* `[>1]` or `[>=2]` to expire at some version (from package.json).
```

Reply:
Agreed

Reply:
Fixed that. Working on the next one.

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/pull/302/commits/4955129775d322f71103a429fd702ca3b5ede7be